_data/simple-icons.json

```diff
@@ -2500,6 +2500,11 @@
             "source": "https://brand.burton.com/logo",
             "guidelines": "https://brand.burton.com/logo"
         },
+        {
+            "title": "Buttondown",
+            "hex": "0069FF",
+            "source": "https://www.buttondown.email"
```

Comment:
Where is this SVG coming from? I'm not seeing any SVGs on the website itself - only PNGs there.

Reply:
https://docs.buttondown.email/icon.svg!

Reply:
Ah, I see - the SVG is used as a mask in the CSS.
See https://docs.buttondown.email/registration-and-setup for a clear example - the icon shows after any links to the app.

I'd argue the source should be docs.buttondown.email in that case :)